### PR TITLE
add tests for files module

### PR DESCRIPTION
**Pull request by sheb**

### CODE REVIEW

File: tests/unit/test_files.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+
+@pytest.mark.parametrize("isfile,expected", [(True, ["file.sql"]), (False, [])])
+def test_list_files(mocker, isfile, expected):
+    mocker.patch("os.listdir", return_value=["file.sql"])
+    mocker.patch("os.path.isfile", return_value=isfile)
+
+    values = files.list_files("tests/test_data/sql/fixtures")
+
+    assert values == expected
```
Comment (k4nar) [inline tests/unit/test_files.py]: Why using mocks instead of using our fixtures for those tests?

**Comment by ewjoachim** [comment]
Still no coverage report ?

File: tests/unit/test_files.py (hunk 1 above)
Comment (sheb) [reply tests/unit/test_files.py]: We wanted to avoid any io operation in our unit test suite.

**Comment by sheb** [comment]
@ewjoachim [this one](https://github.com/peopledoc/west/pull/17) has a report, very strange.

**Comment by sheb** [comment]
rebased on master

**Comment by sheb** [comment]
rebased on master

**Pull request merged**